packages/html_of_jsx/html_of_jsx.0.0.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Render HTML writing JSX"
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/davesnx/html_of_jsx"
+bug-reports: "https://github.com/davesnx/html_of_jsx/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.0.0"}
+  "reason" {>= "3.10.0"}
+  "ppxlib" {> "0.23.0"}
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {= "0.26.1" & with-dev-setup}
```

Comment:
```suggestion
  "ocamlformat" {= "0.26.1" & with-test}
```
This is needed:
```
#=== ERROR while compiling html_of_jsx.0.0.2 ==================================#
# context              2.2.0~beta3~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/davesnx/html_of_jsx/releases/download/0.0.2/html_of_jsx-0.0.2.tbz)
# path                 ~/.opam/5.0/.opam-switch/build/html_of_jsx.0.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p html_of_jsx -j 39 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/html_of_jsx-7-4b11e0.env
# output-file          ~/.opam/log/html_of_jsx-7-4b11e0.out
### output ###
# File "test/cram/dune", line 3, characters 53-71:
# 3 |  (deps ppx.sh %{bin:jsx-ppx-standalone} %{bin:refmt} %{bin:ocamlformat}))
#                                                          ^^^^^^^^^^^^^^^^^^
# Error: Program ocamlformat not found in the tree or in PATH
```

This type of tests is brittle, I would recommend to avoid them or only have them in the project ci in the future